operations/reservations.md

```diff
@@ -694,6 +695,7 @@ Updates information about a reservation. Note that if any of the fields are sent
 | `ClientToken` | string | required | Token identifying the client application. |
 | `AccessToken` | string | required | Access token of the client application. |
 | `ReservationId` | string | required | Identifier of the reservation within the transaction. |
+| `Reason` | string | required | Reason for updating the reservation. |
```

Comment:
@Alx-Ross The reason is required only when the pricing is being updated.

Reply:
Fixed